boards/lilygo/ttgo_t7v1_5/Kconfig.defconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright 2024 Tobias Kässer <t.kaesser@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_TTGO_T7V1_5
+	select SOC_ESP32
```

Comment:
this does not go in this file, it should be `Kconfig.ttgo_t7v1_5`

Reply:
done